disperser/apiserver/server.go

```diff
@@ -521,6 +521,7 @@ func (s *DispersalServer) GetBlobStatus(ctx context.Context, req *pb.BlobStatusR
 	metadata, err := s.blobStore.GetBlobMetadata(ctx, metadataKey)
 	if err != nil {
 		if errors.Is(err, disperser.ErrMetadataNotFound) {
+			s.metrics.HandleInvalidArgRequest("GetBlobStatus")
 			return nil, api.NewInvalidArgError("no metadata found for the requestID")
```

Comment:
We should use this error: https://github.com/Layr-Labs/eigenda/blob/master/api/errors.go#L27